linkerd/docs/namer.md

```diff
@@ -318,6 +318,123 @@ port-name | yes | The port name.
 svc-name | yes | The name of the service.
 label-value | yes if `labelSelector` is defined | The label value used to filter services.
 
+### K8s Namespaced Configuration
+
+> Example usage of the namespaced namer that routes traffic to services within the current namespace
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: l5d-config
+data:
+  config.yaml: |-
+    namers:
+    - kind: io.l5d.k8s.ns
+      host: localhost
+      port: 8001
+      envVar: MY_POD_NAMESPACE
+
+    routers:
+    - protocol: http
+      dtab: |
+        /svc => /#/io.l5d.k8s.ns/admin;
+      servers:
+      - port: 4140
+        ip: 0.0.0.0
+
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: l5d
+  name: l5d
+spec:
+  template:
+    metadata:
+      labels:
+        app: l5d
+    spec:
+      volumes:
+      - name: l5d-config
+        configMap:
+          name: "l5d-config"
+      containers:
+      - name: l5d
+        image: buoyantio/linkerd:1.0.0-SNAPSHOT
```

Comment:
what do you think about using the `latest` tag so that we don't have to update this for each version bump?